### PR TITLE
Add a way to navigate to the home page and start over

### PR DESCRIPTION
**Pull request by microbit-robert**
See https://microbit-global.monday.com/boards/1356069004/views/10076885/pulses/1356382517

### CODE REVIEW

**Comment by microbit-robert** [comment]
Preview: https://homepage-nav.ml-trainer.pages.dev/

**Comment by microbit-robert** [comment]
This is just waiting for copy approval.

File: src/pages/Homepage.svelte
Comment (microbit-robert) [reply src/pages/Homepage.svelte:45]: Hah, good catch. I guess it will have to be a mixture of both some name or some recordings. Will fix tomorrow.

Comment (microbit-robert) [reply src/pages/Homepage.svelte]: [0d17f9f](https://github.com/microbit-foundation/ml-trainer/pull/142/commits/0d17f9f58ee77680d9a3e879f77a14d74968a3a5)

**Review by microbit-grace — APPROVED** [review]
Thanks, LGTM

**Pull request merged**
